terraform/runner-manager/set_space_egress.sh

```diff
@@ -50,11 +50,11 @@ if [[ $space = "" ]]; then
 fi
 
 if [[ $public = true ]]; then
-  cf bind-security-group public_networks_egress $org --space $space > /dev/null
+  cf bind-security-group public_networks_egress "$org" --space "$space" > /dev/null
 fi
 
 if [[ $trusted = true ]]; then
-  cf bind-security-group trusted_local_networks_egress $org --space $space > /dev/null
+  cf bind-security-group trusted_local_networks_egress "$org" --space "$space" > /dev/null
```

Comment:
Praise (non-blocking... sitting here giggling at the thought of blocking praise): "Thank you!"